v1/onthisday.yaml

```diff
@@ -16,7 +16,7 @@ paths:
          - Selected: Featured, most important events
          - All: Everything that happened on the provided day of the year
 
-        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#Unstable)
```

Comment:
Agreed. The iOS app hasn't even started using it. It's likely that there would be change requests once it's actually used by the iOS app.